PhysicsTools/PatAlgos/plugins/PATTauProducer.cc

```diff
@@ -315,6 +316,11 @@ void PATTauProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSetup
 	  edm::Handle<reco::PFTauDiscriminator> pfTauIdDiscr;
 	  iEvent.getByToken(pfTauIDTokens_[i], pfTauIdDiscr);
 
+	  if(skipMissingTauID_ && !pfTauIdDiscr.isValid()){
+	    edm::LogWarning("DataSource") << "Tau discriminator '" << tauIDSrcs_[i].first
```

Comment:
Can this be made a LogInfo?
The warning shows up for every tau and just pollutes the logs.
A more informative option that will still be printed is to accumulate the number of dropped IDs and print only the first time it happens.